srml/contract/src/tests.rs

```diff
@@ -78,6 +78,15 @@ impl balances::Trait for Test {
 	type EnsureAccountLiquid = ();
 	type Event = MetaEvent;
 }
+impl timestamp::Trait for Test {
+	type Moment = u64;
+	type OnTimestampSet = ();
+}
+impl consensus::Trait for Test {
```

Comment:
Why do we need this one?

Reply:
timestamp::Trait depends on it to be implemented